packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -1,39 +1,59 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tile-background-color: Specifies the background color of the component.
+ * @prop --calcite-tile-border-color: Specifies the border color of the component.
+ * @prop --calcite-tile-border-color-active: Specifies the border color of the component when in an active state.
+ * @prop --calcite-tile-border-color-hover: Specifies the border color of the component when in a hover state.
+ * @prop --calcite-tile-description-text-color: Specifies the description text color of the component.
+ * @prop --calcite-tile-description-text-color-hover: Specifies the description text color of the component when in an active or hover state.
+ * @prop --calcite-tile-heading-text-color: Specifies the heading text color of the component.
+ * @prop --calcite-tile-heading-text-color-hover: Specifies the heading text color of the component when in an active or hover state.
+ * @prop --calcite-tile-icon-color: Specifies the icon color of the component.
+ */
+
 :host {
-  @apply bg-foreground-1
-    text-color-3
-    box-border
-    inline-block
-    select-none
-    duration-150
-    ease-in-out;
-
-  .container {
-    @apply pointer-events-none
-      grid
-      grid-cols-1;
-    gap: var(--calcite-spacing-md);
-  }
+  --calcite-tile-background-color: var(--calcite-color-foreground-1);
+  --calcite-tile-border-color: var(--calcite-color-border-2);
+  --calcite-tile-border-color-active: var(--calcite-color-brand);
+  --calcite-tile-border-color-hover: var(--calcite-color-brand);
+  --calcite-tile-description-text-color: var(--calcite-color-text-3);
+  --calcite-tile-description-text-color-hover: var(--calcite-color-text-2);
+  --calcite-tile-heading-text-color: var(--calcite-color-text-2);
+  --calcite-tile-heading-text-color-hover: var(--calcite-color-text-1);
+  --calcite-tile-icon-color: var(--calcite-color-text-3);
+
+  box-sizing: border-box;
+  display: inline-block;
+  max-inline-size: 460px;
```

Comment:
where are these pixel sizes coming from?

Reply:
From the [Figma specs](https://www.figma.com/file/8CiAVHkXIMTsI9hTSbotEz/Tile-Redesign-%5BEpic-6662%5D?type=design&node-id=1107-2810&mode=dev):

![image](https://github.com/Esri/calcite-design-system/assets/821864/93480c6c-ff98-4d0f-bf10-5c582aa075f4)

Reply:
@SkyeSeitz should these be container tokens or based off container tokens?

Reply:
@alisonailea great question. The ones highlighted in blue below are the only values with something close in our container tokens. Do you think we should add the rest as container tokens? 

<img width="713" alt="image" src="https://github.com/Esri/calcite-design-system/assets/29716057/2efc3b96-dd04-49b8-984b-9d4e2d8d6162">